utils/tzif/src/lib.rs

```diff
@@ -40,11 +38,10 @@ pub mod parse;
 /// Error types an implementations.
 pub mod error;
 
-/// Parses a `TZif` file at the provided `path`.
-pub fn parse_tzif_file(path: &Path) -> Result<TzifData, Error> {
-    let file = File::open(path)?;
+/// Parses a `TZif` file.
+pub fn parse_tzif(data: &[u8]) -> Result<TzifData, Error> {
```

Comment:
Definitely think this is better. I ran into not being able to read bytes and though it was weird. But if it's a change to main functionality. Would it be worth having an example for how to read from a file? In case someone is using `tzif` elsewhere.